cmd/rook/userfacing/multus/validation/validation.go

```diff
@@ -184,6 +191,11 @@ func runValidation(ctx context.Context) {
 		}
 	}
 
+	// allow --host-check-only(=true) flag to override default/configfile settings
```

Comment:
If the default were false, this override wouldn't work. Can we differentiate between the CLI param being specified or not? But maybe it's not worth worrying about.

Reply:
I'm not sure I understand. The default is false.